.github/workflows/rbc_test.yml

```diff
@@ -256,8 +190,20 @@ jobs:
           mamba run -n docker docker-compose logs --no-color --tail=10000 -f -t \> omniscidb-docker.log
           cat omniscidb-docker.log
 
-      - name: Show Omniscidb server output [conda]
+      - name: Stop Omniscidb server [docker]
         shell: bash -l {0}
-        if: matrix.os == 'ubuntu-latest' && matrix.omniscidb-from == 'conda'
+        timeout-minutes: 1
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.omniscidb-from == 'docker' }}
         run: |
-          cat omniscidb-output.txt
+          mamba run -n docker docker-compose stop
+          sleep 10
+
+      - name: Run RemoteJIT tests
```

Comment:
Does this mean that RemoteJIT tests are run 3 times for each omniscidb version?

It seems that this change disables testing RemoteJIT under Windows and MacOS. Can you restore it? If the matrix grows two big, then restore at least some combinations so that we know that rbc is functional under non-Linux OSs as well.

Reply:
Yep, just restored it